<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PowerCableForm</class>
 <widget class="QWidget" name="PowerCableForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>850</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>850</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1920</width>
    <height>1080</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>动力电缆计算器</string>
  </property>
  <widget class="QLabel" name="lblI07">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei UI</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>电缆芯数排列</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblI03">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei UI</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>导体</string>
   </property>
   <property name="buddy">
    <cstring>cmbConductor</cstring>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineECS">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>70</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei UI</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>300</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblI04">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>70</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei UI</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>标称截面积(㎜²)</string>
   </property>
   <property name="buddy">
    <cstring>lineECS</cstring>
   </property>
  </widget>
  <widget class="QLabel" name="lblI02">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei UI</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>发动机单绕组电流(A)</string>
   </property>
   <property name="buddy">
    <cstring>lineEEC</cstring>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEEC">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>40</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei UI</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>1600</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QComboBox" name="cmbPCType">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>10</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei UI</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="currentText">
    <string>机侧动力电缆</string>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <item>
    <property name="text">
     <string>机侧动力电缆</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>网侧动力电缆</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>地埋电缆</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="lblI01">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei UI</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>电缆类型</string>
   </property>
   <property name="buddy">
    <cstring>cmbPCType</cstring>
   </property>
  </widget>
  <widget class="QComboBox" name="cmbConductor">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>70</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei UI</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <item>
    <property name="text">
     <string>铜</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>铝</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="lblI05">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei UI</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>绝缘材料</string>
   </property>
   <property name="buddy">
    <cstring>cmbMaterial</cstring>
   </property>
  </widget>
  <widget class="QComboBox" name="cmbMaterial">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>100</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei UI</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <item>
    <property name="text">
     <string>PVC</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>XLPE/EPR</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>矿物绝缘</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="lblI06">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>100</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei UI</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>护套耐温(°C)</string>
   </property>
   <property name="buddy">
    <cstring>lineESTR</cstring>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineESTR">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>100</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei UI</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>90</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QComboBox" name="cmbCrossType">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>160</y>
     <width>241</width>
     <height>64</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>53</width>
     <height>64</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>64</height>
    </size>
   </property>
   <property name="baseSize">
    <size>
     <width>0</width>
     <height>64</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei UI</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="iconSize">
    <size>
     <width>53</width>
     <height>64</height>
    </size>
   </property>
   <item>
    <property name="text">
     <string>多芯电缆
(二芯负荷导体)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>多芯电缆
(三芯负荷导体)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>单芯电缆
(相互接触的二根负荷导体)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>单芯电缆
(相互接触的二根负荷导体)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>单芯电缆
(扁平敷设相互接触的三根负荷导体)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>单芯电缆
(扁平敷设的水平有间距的三根负荷导体)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>单芯电缆
(扁平敷设的垂直有间距的三根负荷导体)</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="lblI08">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei UI</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>绕组数</string>
   </property>
   <property name="buddy">
    <cstring>lineEWings</cstring>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEWings">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>130</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei UI</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>2</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblI09">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>10</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei UI</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>敷设方式</string>
   </property>
   <property name="buddy">
    <cstring>cmbLayingType</cstring>
   </property>
  </widget>
  <widget class="QLabel" name="lblI10">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>130</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei UI</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>敷设温度(°C)</string>
   </property>
   <property name="buddy">
    <cstring>lineEAmbientT</cstring>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEAmbientT">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>130</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei UI</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>30</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QComboBox" name="cmbLayingType">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>10</y>
     <width>181</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei UI</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <item>
    <property name="text">
     <string>水平安装的有孔托盘</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>垂直安装的有孔托盘</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>梯架和线夹等</string>
    </property>
   </item>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>40</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei UI</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QRadioButton" name="rbSpace">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei UI</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>有间距</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rbTouch">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei UI</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>相互接触</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QLineEdit" name="lineENumber">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>100</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei UI</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>1</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblI11">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>100</y>
     <width>80</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei UI</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>托盘/梯架数</string>
   </property>
   <property name="buddy">
    <cstring>lineENumber</cstring>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineECircuits">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>130</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei UI</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>1</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblI12">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>130</y>
     <width>80</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei UI</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>三相回路数</string>
   </property>
   <property name="buddy">
    <cstring>lineECircuits</cstring>
   </property>
  </widget>
  <widget class="QLabel" name="lblO01">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>50</y>
     <width>120</width>
     <height>13</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei UI</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>电缆类型</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblO3">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>110</y>
     <width>120</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei UI</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>电缆型号</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblO02">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>70</y>
     <width>120</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei UI</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>变流器单绕组电流</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lblO5">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>160</y>
     <width>120</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei UI</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>敷设温度</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblO4">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>140</y>
     <width>120</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei UI</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>绕组数</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblO06">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>190</y>
     <width>120</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei UI</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>排布方式</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblOPCType">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>50</y>
     <width>100</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei UI</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>机侧动力电缆</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblOEC">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>80</y>
     <width>100</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei UI</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>1600A</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblOPCM">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>110</y>
     <width>100</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei UI</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>300方 90° 铜</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblOWings">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>130</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei UI</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>2</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblOAmbientT">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>160</y>
     <width>100</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei UI</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>30°</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblArrangeType">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>180</y>
     <width>100</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei UI</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>有孔托盘相互接触(1层1回路)</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTableWidget" name="tblList">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>831</width>
     <height>211</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei UI</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <column>
    <property name="text">
     <string>电缆类型</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>发动机单绕组电流(A)</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>导体</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>标称横截面积(㎜²)</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>绝缘材料</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>护套耐温(°C)</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>绕组数</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>敷设温度(°C)</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>电缆芯数排列</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>敷设方式</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>是否接触</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>托盘/梯架数</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>三相回路数</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="btnCalculation">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>10</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei UI</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>试算输出</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnAdd">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>230</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei UI</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>加入列表</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnUpdate">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>230</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei UI</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>更新选中行</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnExport">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>230</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei UI</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>清空列表</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnDelete">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>230</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei UI</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>删除选中行</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnMDMT">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>230</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei UI</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>导出文件</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblLayingImg">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>40</y>
     <width>111</width>
     <height>113</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>113</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>113</height>
    </size>
   </property>
   <property name="baseSize">
    <size>
     <width>0</width>
     <height>113</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei UI</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>cmbPCType</tabstop>
  <tabstop>lineEEC</tabstop>
  <tabstop>cmbConductor</tabstop>
  <tabstop>lineECS</tabstop>
  <tabstop>cmbMaterial</tabstop>
  <tabstop>lineESTR</tabstop>
  <tabstop>lineEWings</tabstop>
  <tabstop>cmbCrossType</tabstop>
  <tabstop>cmbLayingType</tabstop>
  <tabstop>lineEAmbientT</tabstop>
  <tabstop>rbTouch</tabstop>
  <tabstop>rbSpace</tabstop>
  <tabstop>lineENumber</tabstop>
  <tabstop>lineECircuits</tabstop>
  <tabstop>btnCalculation</tabstop>
  <tabstop>btnAdd</tabstop>
  <tabstop>btnUpdate</tabstop>
  <tabstop>btnDelete</tabstop>
  <tabstop>btnExport</tabstop>
  <tabstop>btnMDMT</tabstop>
  <tabstop>tblList</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
